packages/compute-baseline/src/browser-compat-data/release.test.ts

```diff
@@ -87,4 +87,30 @@ describe("Release", function () {
       assert.equal(safariPreview.isPrerelease(), true);
     });
   });
+
+  describe("inRange()", function () {
```

Comment:
Since the API allows for comparing versions from different browsers, can you add a test showing what it does? Throws an error?

Reply:
Done. There's now a test showing that this throws when you compare across browsers.